docs/setup-deploy-toml.adoc

```diff
@@ -0,0 +1,37 @@
+= Deploy manually with a configuration file
+
+If the other deployment methods do not suit your needs, you may choose to run or deploy the agent manually.
+
+1.
+Send us an e-mail at support@hercules-ci.com. We can give advice and
+we are interested in expanding our officially supported deployment methods.
+
+2.
+Write an `agent.toml` file. Most of the entries are optional. A small example:
+
+[source,toml]
+----
+include::agent-small-example.toml[]
+----
+
+This guide assumes that the link:#basedirectory[`baseDirectory`] is set to `/var/lib/hercules-ci-agent`.
+
+3.
+Get a cluster join token.
+
+--
+- In the https://hercules-ci.com/dashboard[dashboard], find the account for which you would like to deploy the agent,
+- Click the "Agents" button and the button in "Generate token" tab. This produces a private token that should be protected like a password.
+- Copy the token into a plain text file `/var/lib/hercules-ci-agent/secrets/cluster-join-token.key`.
+--
+
+4.
+Write a link:#binarycachespath[binary caches file] and install it in `/var/lib/hercules-ci-agent/secrets/binary-caches.json`.
+
+5.
```

Comment:
Maybe skip binary caches for now, as this oversimplifies all the steps needed.

Reply:
I've only moved this file. We can change it, but not in this PR.

Reply:
It's a new section `Deploy manually with a configuration file` that didn't exist before (it was just NixOps).

Reply:
It's this one, modulo `leveloffset`.

Deployed
https://docs.hercules-ci.com/#deploy-manually-with-a-configuration-file

Diff source
https://github.com/hercules-ci/docs.hercules-ci.com/pull/34/files#diff-3d3d28773f27f8a1b52461e6691782feL83

Reply:
https://github.com/hercules-ci/docs.hercules-ci.com/issues/37